packages/react-examples/src/react-badge/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Badge, BadgeProps } from '@fluentui/react-badge';
+
+const BadgeAppearanceExamples = (props: BadgeProps) => (
+  <>
+    <h2>Circular (default)</h2>
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+      }}
+    >
+      <Badge size="smallest" />
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/108070016-529d1980-7064-11eb-9acc-6728003c914a.png)

Can we set a proper font in styles for now? (this should be a part of styles that are set by `Provider`, but we don't have them yet).

Reply:
You mean font family? Do we have it ? ( file or url )?

Reply:
Yup, like in `Avatar` for now:

```tsx
      fontFamily: tokens.global.type.fontFamilies.base,
```

Reply:
Done